_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction
+Wasting a lot of time on building a project is a programming problem. At the moment a programmer starts an
```

Comment:
@Yanich96 

> Wasting a lot of time on building a project is a programming problem. At the moment a programmer starts an assembly, he loses focus on a task and spends valuable working 

"Empty words". We can remove them without losing any meaning.

Reply:
@Yanich96 

> Different build systems use many tools,
 helping to assemble a project faster, namely caching, task parallelization, distributed building and much more.

Why do I need this information?

Reply:
@volodya-lombrozo I written it to start this blog. I will delete these suggestions if they are not necessary.